Reachability/Monitor.swift

```diff
@@ -136,33 +136,33 @@ public class Monitor {
     public init?(withURL URL: NSURL) {
         guard let host = URL.host,
               let reachabilityReference = SCNetworkReachabilityCreateWithName(nil, host) else {
-            return nil;
+            return nil
         }
         
         self.reachabilityReference = reachabilityReference
     }
 
     deinit {
-        self.reachabilityReference = nil;
-        self.unsafeSelfPointer.dealloc(1)
+        reachabilityReference = nil
+        unsafeSelfPointer.deallocate(capacity: 1)
     }
     
     /// Starts the asynchronous monitoring of network reachability.
     ///
     /// - return: true if the notifications started successfully
-    public func startMonitoring() -> Bool {
-        guard !self.monitoringStarted else {
+    public func start() -> Bool {
```

Comment:
Is there a way to know why this would fail? If so, would `throws` make more sense than a info-less `Bool`?